### DISCUSSION TITLE
Currently Working

### DISCUSSION THREAD

**Discussion by Mangesh1511** [ISTE-PCCoE-Web/frontend#11] [General]
Mention the issues you are working on except the figma designs

1) your_github_username-->issue_name

**Comment by Abhijeet-Bhushari** [comment]
I am going to start working on the events page
Abhijeet-Bhushari-->events_page

**Comment by PR3MM** [comment]
PR3MM-->Update Readme #14

**Comment by Mastermind730** [comment]
Mastermind730--> Update Readme adding docs for 3d Assets required for website
